clients/drcachesim/tools/opcode_mix.h

```diff
@@ -103,6 +103,7 @@ class opcode_mix_t : public analysis_tool_t {
         worker_data_t *worker;
         int64_t instr_count;
         std::unordered_map<int, int64_t> opcode_counts;
+        std::unordered_map<uint, int64_t> category_counts;
         std::string error;
         app_pc last_trace_module_start;
         size_t last_trace_module_size;
```

Comment:
Please add a test.  Maybe something like:
```
      torunonly_drcacheoff(opcode_categories allasm_x86_64
        "" "@-simulator_type@opcode_mix" "")
```
That tiny app has a small fixed instr sequence so you can hardcode the category output into a template file `clients/drcachesim/tests/offline-opcode_categories.templatex`.

Reply:
Done.